src/efi.rs

```diff
@@ -381,7 +397,7 @@ impl Component for Efi {
             .run()?;
         if update_firmware {
             if let Some(vendordir) = self.get_efi_vendor(&Path::new(src_root))? {
-                self.update_firmware(device, destd, &vendordir)?
+                self.update_firmware(device, destd, &vendordir)?;
```

Comment:
Hum, this looks weird as we did not change the return signature of this function.